user_sync/config.py

```diff
@@ -233,7 +230,7 @@ def get_dict_from_sources(self, sources, owner):
             if (isinstance(source, types.StringTypes)):
                 absolute_path = self.get_absolute_file_path(source)
```

Comment:
This is where you should be calling `ConfigFileLoader.get_relative_filename` on the `source`, rather than the `get_absolute_file_path` that's being called.  But it might be easier just to change self.get_absolute_file_path to do the call to ConfigFileLoader.